tests/exercises/d2e3.test.js

```diff
@@ -28,25 +28,21 @@ describe('Alarms API endpoints', function () {
     )
   })
 
-  it('GET /alarms/:index should return a specific alarm', function () {
-    request
+  it('GET /alarms/:index should return a specific alarm', async function () {
+    const response = await request
       .get('/alarms/0')
       .expect('Content-Type', /json/)
       .expect(200)
-      .end((err, res) => {
-        if (err) return done(err)
-        assert.strictEqual(
-          res.body.time,
-          '08:30',
-          'Should return the first alarm'
-        )
-        assert.strictEqual(
-          res.body.message,
-          'Wake up!',
-          'Should return the correct message'
-        )
-        done()
-      })
+    assert.strictEqual(
+      response.body.time,
+      '08:30',
+      'Should return the first alarm'
+    )
+    assert.strictEqual(
+      response.body.message,
+      'Wake up!',
+      'Should return the correct message'
+    )
```

Comment:
Rewrote this to use async rather than done syntax to be consistent to the rest of the test file